src/flat-button.jsx

```diff
@@ -85,12 +148,16 @@ const FlatButton = React.createClass({
 
   getDefaultProps() {
     return {
+      disabled: false,
       labelStyle: {},
-      labelPosition: 'before', // Should be after but we keep it like for now (prevent breaking changes)
+      // labelPosition Should be after but we keep it like for now (prevent breaking changes)
+      labelPosition: 'before',
```

Comment:
honestly, I think we should deprecate this pattern throughout the library. I mean, we don't need label OR children! we can just check if children is string. Wouldn't you agree?